src/holofuel/components/layout/PrimaryLayout/PrimaryLayout.js

```diff
@@ -37,7 +43,7 @@ export function PrimaryLayout ({
       agent={holofuelUser}
       agentLoading={holofuelUserLoading}
       inboxCount={inboxCount}
-      holofuelBalance={holofuelBalance}
+      holofuelBalance={ledgerNullState || holofuelBalance}
```

Comment:
I think this could be simpler and easier to read. How about just passing `ledgerLoading` as a separate prop to `SideMenu`, and then in `SideMenu` you can branch on that, rather than testing if `holofuelBalance` is `NaN`.